core/src/main/java/org/apache/accumulo/core/util/shell/commands/OptUtil.java

```diff
@@ -117,7 +117,7 @@ public static OptionGroup addListDeleteGroup(final String name) {
   }
 
   public static Option startRowOpt() {
-    final Option o = new Option(START_ROW_OPT, "begin-row", true, "begin row (inclusive)");
+    final Option o = new Option(START_ROW_OPT, "begin-row", true, "begin row (NOT) inclusive");
```

Comment:
Makes sense. I would say that the Merge command was always wrong (or at least confusing) in its use of parens there.